.github/workflows/test/acceptance-tests/setup_aiven_project.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+show_help() {
+    echo "Usage: $0 -t AIVEN_TOKEN"
+    echo "  -t AIVEN_TOKEN    Set the Aiven API token."
+    echo "  -h                Show help."
+}
+
+while getopts 'ht:' flag; do
+    case "${flag}" in
+        t) AIVEN_TOKEN="${OPTARG}" ;;
+        h) show_help
+           exit 0 ;;
+        *) show_help
+           exit 1 ;;
+    esac
+done
+
+if [ -z "$AIVEN_TOKEN" ]; then
+    echo "Error: Aiven API token is required."
+    show_help
+    exit 1
+fi
+
+response=$(curl -s -H "Authorization: Bearer $AIVEN_TOKEN" https://api.aiven.io/v1/project)
+
+projects=$(echo "$response" | \
+  jq '[.projects[] | select(.project_name | startswith("aiven-ci-terraform-provider"))]')
+
+for project in $(echo "${projects}" | jq -r '.[].project_name'); do
+    services=$(curl -s -H "Authorization: Bearer $AIVEN_TOKEN" \
+      "https://api.aiven.io/v1/project/$project/service")
+    if [ "$(echo "$services" | jq '.services | length')" -eq 0 ]; then
+        echo "Selected project: $project"
+        break
+    fi
```

Comment:
Why this code duplicates `acceptance-test.yml`?

Reply:
this is a script which allows to run the step as an atomic bash script in order to test things locally